test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/CpsPackageReferenceProjectTests.cs

```diff
@@ -4127,6 +4126,107 @@ public async Task GetInstalledAndTransitivePackagesAsync_TransitiveOriginsParamS
             Assert.All(packages.TransitivePackages, pkg => Assert.NotEmpty(pkg.TransitiveOrigins));
         }
 
+        [Theory]
+        [InlineData(false, false)]
+        [InlineData(false, true)]
+        [InlineData(true, false)]
+        [InlineData(true, true)]
+        public async Task GetInstalledAndTransitivePackagesAsync_IsInstalledAndTransitiveComputationNeeded_False_Succeed(bool includeTransitivePackages, bool includeTransitiveOrigins)
```

Comment:
The suffix of this test method is "Succeed" but that doesn't tell me what is supposed to happen. Can you follow the pattern for naming tests here: https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#unit-test-method-naming

The last part should be "with what expectations"